src/CognitiveServices/CognitiveServices/help/Get-AzCognitiveServicesCommitmentPlan.md

```diff
@@ -0,0 +1,111 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CognitiveServices.dll-Help.xml
+Module Name: Az.CognitiveServices
+online version: https://learn.microsoft.com/powershell/module/az.cognitiveservices/get-azcognitiveservicescommitmentplan
+schema: 2.0.0
+---
+
+# Get-AzCognitiveServicesCommitmentPlan
+
+## SYNOPSIS
+Get Cognitive Services Commitment Plan
+
+## SYNTAX
+
+### ResourceGroupParameterSet
+```
+Get-AzCognitiveServicesCommitmentPlan [[-ResourceGroupName] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### AccountNameParameterSet
+```
+Get-AzCognitiveServicesCommitmentPlan [-ResourceGroupName] <String> [-Name] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get Cognitive Services Commitment Plan
+
+## EXAMPLES
+
+### Example 1
+```powershell
+Get-AzCognitiveServicesCommitmentPlan -ResourceGroupName ResourceGroupName -Name CommitmentPlanName
+```
+
```

Comment:
Add output for each example with following pattern please 
\```output
{output}
\```

Reply:
My understanding is that this is optional right?
For existing commands, we didn't add it. I think it's better we find another time and add those together.

Reply:
sure. hope this work item can be tracked in somewhere.